pom.xml

```diff
@@ -64,6 +64,7 @@
     <!-- Be careful to set this as a provided dep, so that it doesn't interfere with dependencies
          from other projects. For example, cassandra and spring boot set guava versions -->
     <guava.version>19.0</guava.version>
+    <netflix.concurrency.limits.version>0.0.49</netflix.concurrency.limits.version>
```

Comment:
@adriancole, concurrency-limits v0.2.0 is the latest tag but uses Java 8 features (such as Optional).  Do you know of any current plans to get io.zipkin.zipkin2:zipkin off of 1.6 and up to at least 1.8?  I saw #777 but it doesn't appear to have actually upped the `main.signature.artifact`.

It also looks like [retrolambda does not support recompiling dependencies](https://github.com/evant/gradle-retrolambda/issues/80) or I'd explore that option further.

Reply:
Never mind, made a new artifact outside of zipkin-core so it's not subject to this limitation.  Affords me some other luxuries around not-shading dependencies as well :)